migrate.py

```diff
@@ -1205,6 +1198,28 @@ def assemble_collections(checkout_path, spec, args, target_github_org):
             REMOVE = set()
 
 
+def rewrite_unit_tests(collection_dir, collection, spec, namespace, args):
+    deps = []
+
+    for file_path in itertools.chain.from_iterable(
+            (os.path.join(dp, f) for f in fn if f.endswith('.py'))
+            for dp, dn, fn in os.walk(os.path.join(collection_dir, 'tests', 'unit'))
+    ):
+        _unit_test_module_src_text, unit_test_module_fst = read_module_txt_n_fst(file_path)
+        deps += rewrite_imports(unit_test_module_fst, collection, spec, namespace, args)
+        deps += rewrite_unit_tests_patch(unit_test_module_fst, collection, spec, namespace, args, file_path)
+        write_text_into_file(file_path, unit_test_module_fst.dumps())
+
+    return deps
+
+
+def add_deps_to_metadata(deps, galaxy_metadata):
+    for dep_ns, dep_coll in deps:
```

Comment:
This can be replaced with 
```suggestion
    new_deps = dict.fromkeys(map('.'.join, deps), '>=1.0')
    galaxy_metadata['dependencies'].update(new_deps)
```